File: src/main/java/no/nav/syfo/domain/Fastlege.java
package no.nav.syfo.domain;


import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class Fastlege {
    public String fornavn;
    public String mellomnavn;
    public String etternavn;
    public String fnr;
    public Integer herId;
    public Integer foreldreEnhetHerId;
    public Integer helsepersonellregisterId;
    public Pasient pasient;
    public Fastlegekontor fastlegekontor;
    public Pasientforhold pasientforhold;
}

File: src/main/kotlin/no/nav/syfo/fastlege/domain/Fastlege.kt
package no.nav.syfo.fastlege.domain

data class Fastlege(
    val fornavn: String,
    val mellomnavn: String,
    val etternavn: String,
    val fnr: String?,
    val herId: Int?,
    val helsepersonellregisterId: Int?,
    val fastlegekontor: Fastlegekontor,
    val pasientforhold: Periode,
    val foreldreEnhetHerId: Int? = null,
    val pasient: Pasient? = null,
    val gyldighet: Periode,
    val relasjon: Relasjon,
    val stillingsprosent: Int?,
)
